drivers/SmartThings/sonos/src/api/sonos_connection.lua

```diff
@@ -186,6 +199,7 @@ function SonosConnection.new(driver, device)
         Router.cleanup_unused_sockets(self.driver)
 
         if not self:coordinator_running() then
+          --TODO this is not infallible
```

Comment:
Is it worth checking this now? My gut is that this should be "eventually consistent" using the automatic reconnect semantics even without it but it's a good call-out.

Reply:
I felt it was too big of a change functionally for what we are trying to accomplish in this PR.